src/occurrence.jl

```diff
@@ -18,21 +18,20 @@ get a few results rapidly for illustration purposes.
 """
 function occurrences()
   occ_s_url = gbifurl * "occurrence/search"
-  occ_s_req = Requests.get(occ_s_url)
+  occ_s_req = HTTP.get(occ_s_url)
   if occ_s_req.status == 200
-    body = Requests.json(occ_s_req)
+    body = JSON.parse(String(occ_s_req.body))
     occ = map(GBIFRecord, body["results"])
     maxocc = body["count"] > 200000 ? 200000 : body["count"]
     return GBIFRecords(
       body["offset"],
       maxocc,
       nothing,
-      view(occ, collect(1:length(occ))),
       occ,
       ones(Bool, length(occ))
     )
   else
-    warn("Non-OK status returned: ", occ_s_req.status)
+    @error "Non-OK status returned: $(occ_s_req.status)"
```

Comment:
Why change from warn to error?

Reply:
It's probably better to stop the analysis if something goes wrong.

Reply:
yeah good point